proposals/lip-0045.md

```diff
@@ -1390,7 +1380,11 @@ Let `genesisBlockAssetBytes` be the `data` bytes included in the block assets fo
   * The `bftWeight` property of each element must be positive integer.
   * Let `totalWeight` be the sum of the `bftWeight` property of every element in `activeValidators`. Then `totalWeight` has to be less than or equal to `MAX_UINT64`.
   * Check that `totalWeight//3 + 1 <= certificateThreshold <= totalWeight`, where `//` indicates integer division.
-* If a chain account for another sidechain is present, than a chain account for the mainchain (`chainID == getMainchainID()`) must be present, as well as the own chain account.
+  * Check that the corresponding `validatorsHash` stored in `chainAccount.lastCertificate.validatorsHash` matches with the value computed from `activeValidators` and `certificateThreshold`.
+* If a chain account for another sidechain is present (`storeKey != getMainchainID()`), then a chain account for the mainchain (`storeKey == getMainchainID()`) must be present, as well as the own chain account.
```

Comment:
This makes sense only for sidechains. State this explicitly.

Also I guess for sidechains the `registeredNamesSubstore` property should be empty or even do not exist in the `genesisInteroperabilityStoreSchema`.

Reply:
added